nftables.go

```diff
@@ -329,6 +329,21 @@ func (cc *Conn) FlushRuleset() {
 	})
 }
 
+// DelTable deletes a specific table, aling with all chains/rules it contains.
+func (cc *Conn) DelTable(t *Table) {
+	data := cc.marshalAttr([]netlink.Attribute{
+		{Type: unix.NFTA_TABLE_NAME, Data: []byte(t.Name + "\x00")},
+		{Type: unix.NFTA_TABLE_FLAGS, Data: []byte{0, 0, 0, 0}},
+	})
+	cc.messages = append(cc.messages, netlink.Message{
+		Header: netlink.Header{
+			Type:  netlink.HeaderType((unix.NFNL_SUBSYS_NFTABLES << 8) | unix.NFT_MSG_DELTABLE),
+			Flags: netlink.HeaderFlagsRequest | netlink.HeaderFlagsAcknowledge | netlink.HeaderFlagsCreate,
```

Comment:
Are you sure about the flags? In libnftnl/examples/nft-table-del.c, I only see HeaderFlagsAcknowledge.

Maybe a test would be worthwhile after all :)

Reply:
I'm pretty certain `NLM_F_REQUEST` is set at a higher level in the library - FWICT the kernel only handles 'request' messages and [rejects all others](https://github.com/torvalds/linux/blob/9a76aba02a37718242d7cdc294f0a3901928aa57/net/netfilter/nfnetlink.c#L378).

Likewise `NLM_F_ACK` indicates that the kernel should respond with error messages (as opposed to just setting BATCH_FAILURE status. We definitely want this behavior for in the future - it would be nice for `Flush()` to properly detect errors and return a useful error structure.

Reply:
I think `HeaderFlagsCreate` is superfluous thou.

Reply:
Thanks for the details. Can you check if it works without HeaderFlagsCreate and update the PR if so please? :)